src/components/rewards-bar/index.test.tsx

```diff
@@ -94,11 +97,11 @@ describe('rewards-bar', () => {
     expect(wrapper.find(TooltipPopup).prop('open')).toBeFalse();
   });
 
-  it('renders the rewards avatar as necessary', function () {
+  it('renders the SettingsMenu as necessary', function () {
     let wrapper = subject({ includeRewardsAvatar: true });
-    expect(wrapper).toHaveElement('Avatar');
+    expect(wrapper).toHaveElement('SettingsMenu');
 
     wrapper.setProps({ includeRewardsAvatar: false });
-    expect(wrapper).not.toHaveElement('Avatar');
+    expect(wrapper).not.toHaveElement('SettingsMenu');
```

Comment:
maybe also add a test for when you click on the settings menu, you should see the avatar with their display name & the logout button?